==25387== Memcheck, a memory error detector
==25387== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==25387== Using Valgrind-3.21.0-d97fed7c3e-20230428 and LibVEX; rerun with -h for copyright info
==25387== Command: ./build/test/unittest/test_main --gtest_filter=SQLParserTest.good_create_index_1
==25387== Parent PID: 20414
==25387== 
--25387-- 
--25387-- Valgrind options:
--25387--    --leak-check=full
--25387--    --show-leak-kinds=all
--25387--    --track-origins=yes
--25387--    --verbose
--25387--    --log-file=valgrind-out.txt
--25387-- Contents of /proc/version:
--25387--   Linux version 6.2.0-34-generic (buildd@lcy02-amd64-025) (x86_64-linux-gnu-gcc-12 (Ubuntu 12.3.0-1ubuntu1~23.04) 12.3.0, GNU ld (GNU Binutils for Ubuntu) 2.40) #34-Ubuntu SMP PREEMPT_DYNAMIC Mon Sep  4 13:06:55 UTC 2023
--25387-- 
--25387-- Arch and hwcaps: AMD64, LittleEndian, amd64-cx16-lzcnt-rdtscp-sse3-ssse3-avx-avx2-bmi-f16c-rdrand-rdseed
--25387-- Page sizes: currently 4096, max supported 4096
--25387-- Valgrind library directory: /snap/valgrind/154/usr/libexec/valgrind
--25387-- Reading syms from /home/shenyushi/Documents/GitHub/infiniflow/infinity/build/test/unittest/test_main
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/ld-linux-x86-64.so.2
--25387--   Considering /usr/lib/debug/.build-id/cc/23bad777ae59d26be0445659cd1c94244d6bc7.debug ..
--25387--   .. build-id is valid
--25387-- Reading syms from /snap/valgrind/154/usr/libexec/valgrind/memcheck-amd64-linux
--25387--    object doesn't have a dynamic symbol table
--25387-- Warning: inlined fn name without absori
--25387-- is shown as UnknownInlinedFun
--25387-- Scheduler: using generic scheduler lock implementation.
--25387-- Reading suppressions file: /snap/valgrind/154/usr/libexec/valgrind/default.supp
==25387== embedded gdbserver: reading from /tmp/vgdb-pipe-from-vgdb-to-25387-by-shenyushi-on-???
==25387== embedded gdbserver: writing to   /tmp/vgdb-pipe-to-vgdb-from-25387-by-shenyushi-on-???
==25387== embedded gdbserver: shared mem   /tmp/vgdb-pipe-shared-mem-vgdb-25387-by-shenyushi-on-???
==25387== 
==25387== TO CONTROL THIS PROCESS USING vgdb (which you probably
==25387== don't want to do, unless you know exactly what you're doing,
==25387== or are doing some strange experiment):
==25387==   /snap/valgrind/154/usr/libexec/valgrind/../../bin/vgdb --pid=25387 ...command...
==25387== 
==25387== TO DEBUG THIS PROCESS USING GDB: start GDB like this
==25387==   /path/to/gdb ./build/test/unittest/test_main
==25387== and then give GDB the following command
==25387==   target remote | /snap/valgrind/154/usr/libexec/valgrind/../../bin/vgdb --pid=25387
==25387== --pid is optional if only one valgrind process is running
==25387== 
--25387-- REDIR: 0x4026310 (ld-linux-x86-64.so.2:strlen) redirected to 0x581ecd5e (vgPlain_amd64_linux_REDIR_FOR_strlen)
--25387-- REDIR: 0x4024ac0 (ld-linux-x86-64.so.2:index) redirected to 0x581ecd78 (vgPlain_amd64_linux_REDIR_FOR_index)
--25387-- Reading syms from /snap/valgrind/154/usr/libexec/valgrind/vgpreload_core-amd64-linux.so
--25387-- Reading syms from /snap/valgrind/154/usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so
==25387== WARNING: new redirection conflicts with existing -- ignoring it
--25387--     old: 0x04026310 (strlen              ) R-> (0000.0) 0x581ecd5e vgPlain_amd64_linux_REDIR_FOR_strlen
--25387--     new: 0x04026310 (strlen              ) R-> (2007.0) 0x04e0b4d0 strlen
--25387-- REDIR: 0x4024cf0 (ld-linux-x86-64.so.2:strcmp) redirected to 0x4e0c700 (strcmp)
--25387-- REDIR: 0x4023f10 (ld-linux-x86-64.so.2:mempcpy) redirected to 0x4e10920 (mempcpy)
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/libgfortran.so.5.0.0
--25387--    object doesn't have a symbol table
--25387-- Reading syms from /usr/lib/llvm-18/lib/libomp.so.5
--25387--    object doesn't have a symbol table
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/libmkl_intel_lp64.so
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/libmkl_intel_thread.so
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/libmkl_core.so
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/libm.so.6
--25387--   Considering /usr/lib/debug/.build-id/d7/09c6fdf502b867187906dfebb04c76a34f2be7.debug ..
--25387--   .. build-id is valid
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/libstdc++.so.6.0.32
--25387--    object doesn't have a symbol table
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/libgcc_s.so.1
--25387--    object doesn't have a symbol table
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/libc.so.6
--25387--   Considering /usr/lib/debug/.build-id/ff/2d8e707625b73b293961a4bc168e373d14a44a.debug ..
--25387--   .. build-id is valid
==25387== WARNING: new redirection conflicts with existing -- ignoring it
--25387--     old: 0x0e54d560 (memalign            ) R-> (1011.0) 0x04e0a2e1 memalign
--25387--     new: 0x0e54d560 (memalign            ) R-> (1017.0) 0x04e0a7c3 aligned_alloc
==25387== WARNING: new redirection conflicts with existing -- ignoring it
--25387--     old: 0x0e54d560 (memalign            ) R-> (1011.0) 0x04e0a2e1 memalign
--25387--     new: 0x0e54d560 (memalign            ) R-> (1017.0) 0x04e0a751 aligned_alloc
==25387== WARNING: new redirection conflicts with existing -- ignoring it
--25387--     old: 0x0e54d560 (memalign            ) R-> (1011.0) 0x04e0a2e1 memalign
--25387--     new: 0x0e54d560 (memalign            ) R-> (1017.0) 0x04e0a7c3 aligned_alloc
==25387== WARNING: new redirection conflicts with existing -- ignoring it
--25387--     old: 0x0e54d560 (memalign            ) R-> (1011.0) 0x04e0a2e1 memalign
--25387--     new: 0x0e54d560 (memalign            ) R-> (1017.0) 0x04e0a751 aligned_alloc
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/libdl.so.2
--25387--   Considering /usr/lib/debug/.build-id/24/86d9745da8ad2cdccf9a0f4ba96a7b68bdf3d5.debug ..
--25387--   .. build-id is valid
--25387-- Reading syms from /usr/lib/x86_64-linux-gnu/libpthread.so.0
--25387-- REDIR: 0x40264b0 (ld-linux-x86-64.so.2:strncmp) redirected to 0x4e0bd90 (strncmp)
--25387-- REDIR: 0xe554390 (libc.so.6:strnlen) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe554420 (libc.so.6:strpbrk) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe5524f0 (libc.so.6:strcmp) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe569a70 (libc.so.6:wcsnlen) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe551600 (libc.so.6:memset) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe5680e0 (libc.so.6:wcslen) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe550ae0 (libc.so.6:memcpy@@GLIBC_2.14) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe567ea0 (libc.so.6:wcschr) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe5523e0 (libc.so.6:index) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe554450 (libc.so.6:rindex) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe567f30 (libc.so.6:wcscmp) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe551850 (libc.so.6:stpncpy) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe568560 (libc.so.6:wmemchr) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe554230 (libc.so.6:strncmp) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe5518c0 (libc.so.6:strcasecmp) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe553740 (libc.so.6:strcspn) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe567fc0 (libc.so.6:wcscpy) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe552360 (libc.so.6:strcat) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe554120 (libc.so.6:strncasecmp_l) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe5509e0 (libc.so.6:bcmp) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe551570 (libc.so.6:memrchr) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe552460 (libc.so.6:strchrnul) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe5536c0 (libc.so.6:strcpy) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe551960 (libc.so.6:strcasecmp_l) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe553ff0 (libc.so.6:strlen) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe5542d0 (libc.so.6:strncpy) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe5685f0 (libc.so.6:wmemcmp) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe5517d0 (libc.so.6:stpcpy) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe551330 (libc.so.6:memmove) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
==25387== Preferring higher priority redirection:
--25387--     old: 0x0e61e2c0 (__memcpy_avx_unalign) R-> (2018.0) 0x04e0dc40 __memcpy_avx_unaligned_erms
--25387--     new: 0x0e61e2c0 (__memcpy_avx_unalign) R-> (2018.1) 0x04e0f900 memmove
--25387-- REDIR: 0xe550950 (libc.so.6:memchr) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe554610 (libc.so.6:strspn) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe551450 (libc.so.6:mempcpy) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe554080 (libc.so.6:strncasecmp) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe551700 (libc.so.6:rawmemchr) redirected to 0x4a006e8 (_vgnU_ifunc_wrapper)
--25387-- REDIR: 0xe622eb0 (libc.so.6:__strrchr_avx2) redirected to 0x4e0ae50 (rindex)
--25387-- REDIR: 0xe621000 (libc.so.6:__strlen_avx2) redirected to 0x4e0b410 (strlen)
--25387-- REDIR: 0xe61db60 (libc.so.6:__memcmp_avx2_movbe) redirected to 0x4e0ef40 (bcmp)
--25387-- REDIR: 0xe54c920 (libc.so.6:malloc) redirected to 0x4e05043 (malloc)
--25387-- REDIR: 0xe61ecc0 (libc.so.6:__memset_avx2_unaligned_erms) redirected to 0x4e0f7f0 (memset)
--25387-- REDIR: 0xe54ddd0 (libc.so.6:calloc) redirected to 0x4e09eaf (calloc)
--25387-- REDIR: 0xe61e2c0 (libc.so.6:__memcpy_avx_unaligned_erms) redirected to 0x4e0f900 (memmove)
--25387-- REDIR: 0xe2f8600 (libstdc++.so.6:operator new(unsigned long)) redirected to 0x4e057b5 (operator new(unsigned long))
--25387-- REDIR: 0xe2f6860 (libstdc++.so.6:operator delete(void*)) redirected to 0x4e07f83 (operator delete(void*))
--25387-- REDIR: 0xe61d8c0 (libc.so.6:__memchr_avx2) redirected to 0x4e0c7a0 (memchr)
--25387-- REDIR: 0xe620850 (libc.so.6:__strcmp_avx2) redirected to 0x4e0c5c0 (strcmp)
--25387-- REDIR: 0xe6220e0 (libc.so.6:__strncmp_avx2) redirected to 0x4e0bb10 (strncmp)
--25387-- REDIR: 0xe620420 (libc.so.6:__strchr_avx2) redirected to 0x4e0b070 (index)
--25387-- REDIR: 0xe61f700 (libc.so.6:__strcasecmp_avx2) redirected to 0x4e0be10 (strcasecmp)
--25387-- REDIR: 0xe620660 (libc.so.6:__strchrnul_avx2) redirected to 0x4e10450 (strchrnul)
--25387-- REDIR: 0xe61e280 (libc.so.6:__mempcpy_avx_unaligned_erms) redirected to 0x4e10560 (mempcpy)
--25387-- REDIR: 0xe2f6870 (libstdc++.so.6:operator delete(void*, unsigned long)) redirected to 0x4e0820f (operator delete(void*, unsigned long))
--25387-- REDIR: 0xe54cc30 (libc.so.6:free) redirected to 0x4e07a19 (free)
--25387-- REDIR: 0xe61ee00 (libc.so.6:__rawmemchr_avx2) redirected to 0x4e10480 (rawmemchr)
--25387-- REDIR: 0xe2f8660 (libstdc++.so.6:operator new[](unsigned long)) redirected to 0x4e06aa5 (operator new[](unsigned long))
--25387-- REDIR: 0xe2f6890 (libstdc++.so.6:operator delete[](void*)) redirected to 0x4e09017 (operator delete[](void*))
--25387-- REDIR: 0xe61f200 (libc.so.6:__stpncpy_avx2) redirected to 0x4e0f6c0 (stpncpy)
--25387-- REDIR: 0xe54cf90 (libc.so.6:realloc) redirected to 0x4e0a0bb (realloc)
==25387== 
==25387== HEAP SUMMARY:
==25387==     in use at exit: 456 bytes in 14 blocks
==25387==   total heap usage: 8,076 allocs, 8,062 frees, 939,020 bytes allocated
==25387== 
==25387== Searching for pointers to 14 not-freed blocks
==25387== Checked 8,489,184 bytes
==25387== 
==25387== 8 bytes in 1 blocks are indirectly lost in loss record 1 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x103540D: std::__new_allocator<infinity::InitParameter*>::allocate(unsigned long, void const*) (new_allocator.h:147)
==25387==    by 0x1035203: allocate (allocator.h:198)
==25387==    by 0x1035203: allocate (alloc_traits.h:482)
==25387==    by 0x1035203: std::_Vector_base<infinity::InitParameter*, std::allocator<infinity::InitParameter*> >::_M_allocate(unsigned long) (stl_vector.h:378)
==25387==    by 0x1034EBD: void std::vector<infinity::InitParameter*, std::allocator<infinity::InitParameter*> >::_M_realloc_insert<infinity::InitParameter* const&>(__gnu_cxx::__normal_iterator<infinity::InitParameter**, std::vector<infinity::InitParameter*, std::allocator<infinity::InitParameter*> > >, infinity::InitParameter* const&) (vector.tcc:459)
==25387==    by 0x102D3FD: std::vector<infinity::InitParameter*, std::allocator<infinity::InitParameter*> >::push_back(infinity::InitParameter* const&) (stl_vector.h:1289)
==25387==    by 0x102824E: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2196)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x921234: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:655)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387== 
==25387== 24 bytes in 1 blocks are definitely lost in loss record 2 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x10281BD: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2191)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x91BA59: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:532)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 24 bytes in 1 blocks are definitely lost in loss record 3 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x10281BD: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2191)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x91CB9F: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:556)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 24 bytes in 1 blocks are definitely lost in loss record 4 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x10281BD: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2191)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x91DCE8: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:581)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 24 bytes in 1 blocks are definitely lost in loss record 5 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x10281BD: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2191)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x91EE31: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:605)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 24 bytes in 1 blocks are definitely lost in loss record 6 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x10281BD: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2191)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x91FF7A: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:629)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 40 bytes in 1 blocks are definitely lost in loss record 7 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x1028136: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2183)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x91BA59: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:532)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 40 bytes in 1 blocks are definitely lost in loss record 8 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x1028063: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2175)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x91CB9F: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:556)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 40 bytes in 1 blocks are definitely lost in loss record 9 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x1028063: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2175)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x91DCE8: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:581)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 40 bytes in 1 blocks are definitely lost in loss record 10 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x1028136: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2183)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x91EE31: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:605)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 40 bytes in 1 blocks are definitely lost in loss record 11 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x1028136: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2183)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x91FF7A: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:629)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 40 bytes in 1 blocks are definitely lost in loss record 12 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x1028136: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2183)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x921234: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:655)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 64 bytes in 1 blocks are indirectly lost in loss record 13 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x10282FD: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2209)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x921234: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:655)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== 96 (24 direct, 72 indirect) bytes in 1 blocks are definitely lost in loss record 14 of 14
==25387==    at 0x4E05823: operator new(unsigned long) (vg_replace_malloc.c:472)
==25387==    by 0x1028205: sqlparse(void*, infinity::ParserResult*) (src/parser/parser.y:2195)
==25387==    by 0xDFE35A: infinity::SQLParser::Parse(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::shared_ptr<infinity::ParserResult>&) (src/parser/sql_parser.cpp:30)
==25387==    by 0x921234: SQLParserTest_good_create_index_1_Test::TestBody() (test/unittest/parser/sql_parser.cpp:655)
==25387==    by 0x9FE35A: void testing::internal::HandleSehExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9DEB29: void testing::internal::HandleExceptionsInMethodIfSupported<testing::Test, void>(testing::Test*, void (testing::Test::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387==    by 0x9BF552: testing::Test::Run() (third_party/googletest/googletest/src/gtest.cc:2674)
==25387==    by 0x9C011F: testing::TestInfo::Run() (third_party/googletest/googletest/src/gtest.cc:2853)
==25387==    by 0x9C091C: testing::TestSuite::Run() (third_party/googletest/googletest/src/gtest.cc:3012)
==25387==    by 0x9D21F2: testing::internal::UnitTestImpl::RunAllTests() (third_party/googletest/googletest/src/gtest.cc:5870)
==25387==    by 0x9FE7FA: bool testing::internal::HandleSehExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2599)
==25387==    by 0x9E0F49: bool testing::internal::HandleExceptionsInMethodIfSupported<testing::internal::UnitTestImpl, bool>(testing::internal::UnitTestImpl*, bool (testing::internal::UnitTestImpl::*)(), char const*) (third_party/googletest/googletest/src/gtest.cc:2635)
==25387== 
==25387== LEAK SUMMARY:
==25387==    definitely lost: 384 bytes in 12 blocks
==25387==    indirectly lost: 72 bytes in 2 blocks
==25387==      possibly lost: 0 bytes in 0 blocks
==25387==    still reachable: 0 bytes in 0 blocks
==25387==         suppressed: 0 bytes in 0 blocks
==25387== 
==25387== ERROR SUMMARY: 12 errors from 12 contexts (suppressed: 0 from 0)
